src/graph.rs

```diff
@@ -1369,7 +1369,15 @@ impl PyGraph {
     ///       fd.flush()
     ///       graph = rx.PyGraph.read_edge_list(path=path)
     ///   mpl_draw(graph)
-    ///
+    ///   
+    /// For example for csv based data, you may use the below code
```

Comment:
To use a code example with syntax highlighting you either need the double colon or the `.. code-block` directive
```suggestion
    /// For example for csv based data, you may use the below code::
```